### DISCUSSION TITLE
Integers should be interpreted as unix timestamps

### DISCUSSION THREAD

**Discussion by kerinin** [kaskada-ai/kaskada#228] [Ideas]
Per [Wikipedia](https://en.wikipedia.org/wiki/Unix_time):
> Unix time[[a]](https://en.wikipedia.org/wiki/Unix_time#cite_note-5) is a date and time representation widely used in [computing](https://en.wikipedia.org/wiki/Computing). It measures time by the number of [seconds](https://en.wikipedia.org/wiki/Second) that have elapsed since 00:00:00 [UTC](https://en.wikipedia.org/wiki/Coordinated_Universal_Time) on 1 January 1970, the beginning of the Unix [epoch](https://en.wikipedia.org/wiki/Epoch_(computing)), without adjustments made due to [leap seconds](https://en.wikipedia.org/wiki/Leap_second).[[3]](https://en.wikipedia.org/wiki/Unix_time#cite_note-single-unix-spec-4.16-3)

Currently, if you load a CSV containing integers they appear to be interpreted as nanosecon-precision unix timestamps, which isn't a very common interpretation of integers in the data-processing community.

In this same vein, if you load a CSV containing numbers with decimals, they fail to convert to timestamps - we should support interpreting floating point values as unix timestamps as well.

**Comment by bjchambers** [comment]
At least some of this is Arrow default behavior. Are you referring to "in the time column" or in general? I would have expected a *general* column containing integers to be treated as a numeric column, and not convert to nanosecond timestamps or second precision timestamps.
